### PR TITLE
Add podman support

### LINKED ISSUE #363: Add podman support
We need to think if we can make it the default. Or we can default to what;s available on the machine, by priority.

### PR DESCRIPTION
**Pull request by rnc**
Fixes #363

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-Cekit
+CEkit
```
Comment (goldmann) [inline README.rst]: It should be: `CEKit`

File: cekit/cli.py [hunk 2]
```diff
@@ -125,6 +125,21 @@ def build_buildah(ctx, pull, tags):  # pylint: disable=unused-argument
     run_build(ctx, 'buildah')
 
 
+@build.command(name="podman", short_help="Build using Podman engine")
+@click.option('--pull', help="Always try to fetch latest base image.", is_flag=True)
+@click.option('--tag', 'tags', metavar="TAG", help="Use specified tag to tag the image after build, can be specified multiple times.", multiple=True)
+@click.pass_context
+def build_podman(ctx, pull, tags):
```
Comment (goldmann) [inline cekit/cli.py]: You can also add here this:

```suggestion
def build_podman(ctx, pull, tags):   # pylint: disable=unused-argument
```

**Comment by goldmann** [comment]
When it's ready to review, please un-draft this PR so I can merge it.

File: README.rst (hunk 1 above)
Comment (rnc) [reply README.rst]: Darn, thought I caught all of those! ;-)

File: cekit/cli.py (hunk 2 above)
Comment (rnc) [reply cekit/cli.py]: Thanks!

**Comment by goldmann** [comment]
Thanks!

**Pull request merged**